src/EnergyPlus/SingleDuct.cc

```diff
@@ -2099,10 +2099,10 @@ namespace SingleDuct {
 		}
 		if ( Sys( SysNum ).ZoneMinAirFracMethod == ConstantMinFrac ) {
 			if ( ZoneSizingRunDone ) {
-				if ( CurZoneEqNum > 0 ) {
+				if ( CurTermUnitSizingNum > 0 ) {
 					// use the combined defaults or other user inputs stored in DesCoolVolFlowMin
 					if ( Sys( SysNum ).MaxAirVolFlowRate > 0.0 ) {
-						MinAirFlowFracDes = FinalZoneSizing( CurZoneEqNum ).DesCoolVolFlowMin / Sys( SysNum ).MaxAirVolFlowRate;
+						MinAirFlowFracDes = min( 1.0, TermUnitFinalZoneSizing( CurTermUnitSizingNum ).DesCoolVolFlowMin / Sys( SysNum ).MaxAirVolFlowRate );
```

Comment:
Makes sense to me.

Reply:
Why would this ever be > 1 ? There is something else going on.

Reply:
Valid point (why would this ever be > 1?  One example is a hard-sized value for one and autosize for the other that aren't consistent. For the fully autosized case, other fixes in this PR *should* have gotten rid of that, but I haven't confirmed that.